Microsoft.DotNet.Interactive.CSharp/CSharpKernel.cs

```diff
@@ -70,15 +71,17 @@ public CSharpKernel() : base(DefaultKernelName)
 
         public ScriptState ScriptState { get; private set; }
 
-        internal void AddScriptReferences(IReadOnlyList<ResolvedPackageReference> assemblyPaths)
+        public void AddScriptReferences(IReadOnlyList<ResolvedPackageReference> resolvedReferences)
         {
-            var references = assemblyPaths
+            var references = resolvedReferences
                              .SelectMany(r => r.AssemblyPaths)
                              .Select(r => MetadataReference.CreateFromFile(r.FullName));
 
             ScriptOptions = ScriptOptions.AddReferences(references);
         }
 
+        public string ScriptExtension { get { return DefaultScriptExtension; } }
```

Comment:
> public string ScriptExtension { get { return DefaultScriptExtension; } } [](start = 7, length = 73)

This looks like an implementation detail that doesn't need to be on the public interface.